modules/ROOT/pages/index.adoc

```diff
@@ -4,13 +4,17 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: platform, arm, rest, soa, saas, api, proxy, design, develop, anypoint platform, studio, mule, devkit, studio, connectors, auth, exchange, api design, apikit, raml, application network, anypoint, arm, rest, soa, saas, api, proxy
 
-Use the Anypoint Platform to build an application network, a structured set of integrations between apps and services that are secure and easy to reuse.
+Use Anypoint Platform to build an application network, which is a structured network of applications, data, and devices connected by reusable APIs.
```

Comment:
Re "application network, which is a structured network": Should we use a synonym for the second instance of "network" (in "structured network") or reword a bit to avoid repetition of the term in the definition? Perhaps "which is an integrated system of ..."  "integrated web of ...". "integrated structure of ...".

Reply:
How about this (I don't want to weigh down with too much text):  Use Anypoint Platform to build a structured application network of applications, data, and devices connected by reusable APIs.

Reply:
We use "apps" below. "application" --> "app" here?

Reply:
> How about this (I don't want to weigh down with too much text): Use Anypoint Platform to build a structured application network of applications, data, and devices connected by reusable APIs.

"a structured application network of applications" -> "a structured network of apps"

addresses the repetition in "application network of applications"

Reply:
I'd rather not go to apps here.

Reply:
Here's what I'm going with: "Use Anypoint Platform to build a structured application network that connects applications, data, and devices with reusable APIs."

Reply:
Sounds good!